homeworks/vlad.makedon_Vladmaked/homework_6/user-info.js

```diff
@@ -0,0 +1,29 @@
+const user = {};
+
+Object.defineProperties(user, {
+    fullName: {
+        get: () => this.fullName,
+        set: (value) => {
+            this.fullName = value.split('').reduce((accumulator, element, index, array) => (
+                index === 0 || array[index - 1] === ' ' || array[index - 1] === '-'
+                    ? accumulator + element.toUpperCase()
+                    : accumulator + element.toLowerCase()
```

Comment:
it may work, but actually this is a good use case for RegExp, you could just look for first letters in words (`\b\w` for example) and that's it